src/styles/_variables.scss

```diff
@@ -1,25 +1,51 @@
-// Box Design Language Theme Colors
+// Box Design Language Colors, confirm with design before extending
+// @see https://gallery.io/projects/MCHbtQVoQ2HCZTdZ0F79-fQJ/files/MCEJu8Y2hyDScSTYfw7hxWcA-L66cSQvSj4
+
+// Box/Primary Blues
 $bdl-box-blue: #0061d5 !default;
-$bdl-secondary-blue: #003c84 !default;
+$bdl-box-blue-50: #619de5 !default;
```

Comment:
Can't find the usage of this color in this PR might be good to update this on another PR? Will this be include in the release note as something addition?

Reply:
It was already there, few lines below

Reply:
I only added a couple of new ones that I had to use and made explicit about them grays, rest were already in the file.